### PR TITLE
#2299 - Adjust Maximum Allowable Tuition Remittance

### PR DESCRIPTION
**Pull request by sh16011993**
As a part of this PR, the following was done:

- Mandatory fees is incorporated into the allowable maximum (i.e. tuition+books and supplies+mandatory fees.)
- Unit tests were updated

### CODE REVIEW

File: sources/packages/backend/libs/services/src/confirmation-of-enrollment/confirmation-of-enrollment.service.ts [hunk 1]
```diff
@@ -652,7 +655,7 @@ export class ConfirmationOfEnrollmentService {
 
     if (tuitionRemittanceAmount > maxTuitionAllowed) {
       throw new CustomNamedError(
-        "Tuition amount provided should be lesser than both (Actual tuition + Program related costs) and (Canada grants + Canada Loan + BC Loan).",
+        "Tuition amount provided should be lesser than both (Actual tuition + Program related costs + Mandatory fees) and (Canada grants + Canada Loan + BC Loan).",
```
Comment (andrepestana-aot) [inline sources/packages/backend/libs/services/src/confirmation-of-enrollment/confirmation-of-enrollment.service.ts:658]: 👍

Comment (andrepestana-aot) [reply sources/packages/backend/libs/services/src/confirmation-of-enrollment/confirmation-of-enrollment.service.ts:658]: That'd be good to look for other places to add " + Mandatory fees" like in this file: line 639, `confirmation-of-enrollment.institutions.controller`:  line 251.

**Review by andrepestana-aot — COMMENTED** [review]
Great job. Just a comment for you to look for other places to update the comments referring max tuition remittance.

**Review by dheepak-aot — APPROVED** [review]
Looks good. Good job!

**Review by andrepestana-aot — APPROVED** [review]
Looks good!

**Review by lewischen-aot — APPROVED** [review]
LGTM 👍 Please see comments from other reviewers.

**Pull request merged**